noir-projects/noir-protocol-circuits/crates/types/src/abis/kernel_circuit_public_inputs/private_kernel_circuit_public_inputs.nr

```diff
@@ -10,4 +10,5 @@ struct PrivateKernelCircuitPublicInputs {
     validation_requests: ValidationRequests,
     end: PrivateAccumulatedData,
     constants: CombinedConstantData,
+    public_teardown_call_request: CallRequest,
```

Comment:
I'm sure we've already discussed it, but remind me: why didn't we make `public_teardown_call_request` a part of the `PrivateAccumulatedData`...?

Reply:
All good. Because then it would get split off into PublicAccumulatedData, which exists for both `end` and `endNonRevertible`. So it's more like `revert_code` in that it is singular per TX.